src/cloudai/report_generator/comparison_report.py

```
@@ -43,6 +47,31 @@
+    y_axis_label: str
+    chart_type: Literal["line", "bar"] = "line"
+    x_axis_type: Literal["linear", "logarithmic", "category", "indexed_category"] = "logarithmic"
+    x_axis_column: str | None = None
+    x_axis_label: str | None = None
+    y_axis_type: Literal["linear", "logarithmic", "auto"] = "linear"
+
+
+class _IndentedSafeDumper(yaml.SafeDumper):
+    """Indent sequence items beneath their mapping key."""
+
+    def increase_indent(self, flow: bool = False, indentless: bool = False):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `increase_indent`

Add return type annotation: `None`

(ANN202)